# Allow org.bouncycastle imports in the ssl subpackage.
# Required because DefaultSslEngineFactory now uses BouncyCastle PEMParser.
diff --git a/checkstyle/import-control.xml b/checkstyle/import-control.xml
index b1ef62c..f3a3ea3 100644
--- a/checkstyle/import-control.xml
+++ b/checkstyle/import-control.xml
@@ -138,6 +138,7 @@
       </subpackage>
       <subpackage name="ssl">
         <allow pkg="javax.crypto" />
+        <allow pkg="org.bouncycastle" />
       </subpackage>
       <subpackage name="scram">
         <allow pkg="javax.crypto" />
